backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -718,7 +744,13 @@ bool OpenGLSdlGraphicsManager::notifyEvent(const Common::Event &event) {
 	if (event.type != Common::EVENT_CUSTOM_BACKEND_ACTION_START) {
 		return SdlGraphicsManager::notifyEvent(event);
 	}
-
+	if (event.type == Common::EVENT_KEYDOWN) {
```

Comment:
This is not the way to do this.
Please look at how fullscreen switching is done.

I also bet this code has not been tested on with any other renderer than the 2D OpenGL one.
I already suggested you that SdlGraphicsManager is a common ancestor for all SDL graphical backends (2D and 3D).